src/ol-ext/proj/epsg21781projection.js

```diff
@@ -0,0 +1,17 @@
+goog.provide('ol.proj.EPSG21781');
+
+/**
+ * this ol.Map require is a workaround to be sure that ol.proj.common.add()
+ * is called before this projection is added.
```

Comment:
Could you please clarify why we need to call `ol.proj.common.add` before adding the EPSG:21781 projection.